angular.json

```diff
@@ -832,7 +832,7 @@
       "prefix": "cx",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:ng-packagr",
```

Comment:
Converter and Occ.CheckoutState: https://github.com/SAP/spartacus/pull/14165#discussion_r751910463

Reply:
@bgambocjaviniar - I think we've solved this by using `any`?

Reply:
what do you mean?

Reply:
any for now